src/Admin/AdminInterface.php

```diff
@@ -533,6 +533,13 @@ public function configure();
      */
     public function preBatchAction($actionName, ProxyQueryInterface $query, array &$idx, $allElements);
 
+    /**
+     * Return array of default filter parameters.
+     *
+     * @return array<string, mixed>
+     */
+    public function getDefaultFilterParameters();
```

Comment:
You can add this without BC-break.

The strategy is to comment this method with a comment `NEXT_MAJOR: Uncomment this method.`
And in the phpdoc of the class, adding a `@method` line. There are already a lot

Reply:
updated